mautic/mautic
# Merging of grapesjs with history

This should pull in grapesjs mautic3x branch with history into mautic.

To test this PR:

1. Clone the features branch
2. Delete the references to the GrapesJS builder in composer.json and composer.lock so that we do not have the folder on installation
3. Check out this PR
4. Go to plugins and enable GrapesJS (you might need to install plugins depending on how you installed Mautic) and refresh the page to check that it has been enabled successfully
5. Create new email and landing pages - confirm that the new builder is working as expected
